src/app/components/common/node-editor/node-editor.component.ts

```diff
@@ -1078,6 +1082,16 @@ export class NodeEditorComponent implements OnInit {
         });
   }
 
+  getServiceStatus() {
+    if (this.from == 'south') {
+      const service: Service = this.services.find(service => service.name === this.serviceName);
+      return service.status;
+    } else if (this.from == 'north') {
+      const task: NorthTask = this.tasks.find(task => task.name === this.serviceName);
```

Comment:
North can be a service as well.

Reply:
We are good here. In task we are getting service/task from north side.

Reply:
Add inline comment please

Reply:
I will take care of this in other PR.